### PR TITLE
ACS-3227 Simple rules tests.

### PR DESCRIPTION
**Pull request by tpage-alfresco**
Happy path tests for creating and retrieving rules.

### CODE REVIEW

File: packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/CreateRulesTests.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+    /** Check we can create a rule. */
+    @Test (groups = { TestGroup.REST_API, TestGroup.RULES, TestGroup.SANITY })
+    public void createRule()
+    {
+        STEP("Create a folder in existing site");
+        FolderModel folder = dataContent.usingUser(user).usingSite(site).createFolder();
+
+        RestRuleModel ruleModel = new RestRuleModel();
+        ruleModel.setName("ruleName");
```
[the remark this answers is not in the document]
Comment (tpage-alfresco) [reply packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/CreateRulesTests.java:71]: Currently the only option is to give a rule with a name. The action is hard coded to update a property.

[the remark this answers is not in the document]
Comment (tpage-alfresco) [reply packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/CreateRulesTests.java:71]: > I assume these e2e API tests be updated as we refine the "create rule" implementation

Yes - the aim is to add some more tests immediately as a second PR (covering the negative cases and maybe inheritance if that works without more effort). Further tests will be added as the implementation becomes more complete.

Hopefully we can start adding e2e tests in the same PR as features/fixes once we've covered the basics.

**Comment by tpage-alfresco** [comment]
Merging since tests suite that failed on PR build passed on branch build.
